apps/sms/views/conversation/js/recipients.js

```diff
@@ -54,6 +54,7 @@
     // (ignoring the presense of a '+'), the input value
     // is questionable and may be invalid.
     number = this.number[0] === '+' ? this.number.slice(1) : this.number;
+    number = number.replace('(','').replace(')','');
```

Comment:
Hey, you solution works, but I think Julien's suggestion from https://bugzilla.mozilla.org/show_bug.cgi?id=1120572#c7 will be much nicer - could you please follow the suggestion and make it via regex? Your regex should be declared right after ```rdigit``` in this file.